### PR TITLE
New E2E-Tests for data view

### PR DESCRIPTION
**Pull request by Marcelfrueh**
### Purpose
 Adds two new E2E-Tests for data view: Testing functionality of table view and time series view. Validates that new graphs are displayed correctly and that filtering is working.

### Remarks

PR introduces (a) breaking change(s): no

PR introduces (a) deprecation(s): no

### CODE REVIEW

**Comment by Marcelfrueh** [comment]
Unfortunately, both tests are flaky: Changing the filter settings throws the error: ResizeObserver loop completed with undelivered notifications. Errors occur when adding a filter (line 55 in table.spec.ts and line 77 in timeSeriesSave.spec.ts)

**Comment by Marcelfrueh** [comment]
Added 5 new Tests for different widgets in data view: Now testing the functionality of table, time series, 2D-correlation, distribution, heatmap, map and indicator.
To avoid the error message "ResizeObserver loop completed with undelivered notifications" when adding a filter, the test for the filter is excluded from the UI tests.

File: ui/cypress/tests/datalake/widgets/2d-correlation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { DataLakeUtils } from '../../../support/utils/datalake/DataLakeUtils';
+
+describe('Test 2d Correlation View in Data Explorer', () => {
+    beforeEach('Setup Test', () => {
+        cy.initStreamPipesTest();
+        DataLakeUtils.loadRandomDataSetIntoDataLake();
+    });
+
+    it('Perform Test', () => {
+        DataLakeUtils.goToDatalake();
```
Comment (tenthe) [inline ui/cypress/tests/datalake/widgets/2d-correlation.spec.ts:28]: The setup of this test seems to be the same for each of the widget tests. Can you please extract this in a seperate utils file?

File: ui/cypress/tests/datalake/widgets/2d-correlation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        DataLakeUtils.dataConfigSelectAllFields();
+
+        DataLakeUtils.selectVisualizationConfig();
+
+        DataLakeUtils.selectVisualizationType('2D Correlation');
+
+        DataLakeUtils.clickCreateButton();
+
+        DataLakeUtils.selectTimeRange(
```
Comment (tenthe) [inline ui/cypress/tests/datalake/widgets/2d-correlation.spec.ts:44]: Is it possible to add this also to the new utils function? Maybe it is possible to change the time range before adding the widget.

**Review by tenthe — CHANGES_REQUESTED** [review]

**Review by tenthe — CHANGES_REQUESTED** [review]
Thanks a lot for implementing those tests. I had only one comment regarting the setup code for eacht test. This seems to be very similar. Maybe we can introduce a function in the DataLakeUtils class that is used in eacht test for set up.

**Comment by Marcelfrueh** [comment]
Done!

**Review by tenthe — APPROVED** [review]

**Review by dominikriemer — APPROVED** [review]

**Pull request merged**